migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
Typo: identies -> identities
I'd rename the key to `identities_user_id_fk`. That's enough, don't you think?
Also I'd add a `ON DELETE CASCADE`. I know this is not needed since we don't do hard deletes but if we need to do it, then we're safe. So in total, this is what I propose:

```sql
ALTER TABLE identities ADD constraint identities_user_id_fk FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE CASCADE;
```